src/item.c

```diff
@@ -190,6 +190,17 @@ item_make_link (itemPtr item)
 	return link;
 }
 
+gchar *
```

Comment:
I think this should return a `const gchar *`. `metadata_list_get` returns const.